src/App.tsx

```diff
@@ -1,24 +1,173 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import {
+  addTodo, changeTodo, deleteTodo, getTodos,
+} from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Filters } from './types/Filter';
+import { TodoHeader } from './components/TodoHeader';
+import { TodoFooter } from './components/TodoFooter';
+import { TodoError } from './components/TodoError';
+import { UpdateTodoArgs } from './types/UpdateTodoArgs';
+import { filterTodos } from './utils/filterTodos';
 
-const USER_ID = 0;
+const USER_ID = 10908;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<string | null>(null);
```

Comment:
let's create enum for the error messages. Enums are easier to maintain than strings